libs/sdk-ui-filters/src/AttributeFilter/types.ts

```diff
@@ -251,6 +284,11 @@ export interface IAttributeFilterCoreProps {
      * Enables the migration of displayAsLabel to be immediately reported to the parent app.
      */
     enableImmediateAttributeFilterDisplayAsLabelMigration?: boolean;
+
+    /**
+     * Enables the migration of displayAsLabel to be immediately reported to the parent app.
+     */
+    enable?: boolean;
```

Comment:
This is too vague name for this prop.

Reply:
This is some weird copy-paste. Its unused. I will remove it. Thanks for catching it.